web/blog/2023-03-08-building-a-full-stack-app-supabase-vs-wasp.md

```diff
@@ -0,0 +1,365 @@
+---
+title: "Building a full-stack app: Supabase vs. Wasp"
```

Comment:
```suggestion
title: "Building a full-stack app for learning Italian: Supabase vs. Wasp"
```
trying to make a title bit more "real" :D. Another idea: "I built the same app with Supabase and Wasp: here are the differences" - more "influencer" style :D